data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodescutplane.asset

```diff
@@ -0,0 +1,88 @@
+asset.require("spice/base")
+local assetHelper = asset.require("util/asset_helper")
+local transforms = asset.require('scene/solarsystem/sun/transforms_heliosphere')
+--15 cmr emin01, 16 cmr emin03
```

Comment:
I don't understand this comment 🤔 if useful, could it be rewritten to be more clear?